blocks/research-publications-cards/research-publications-cards.js

```diff
@@ -0,0 +1,54 @@
+import {
+  div, a,
+} from '../../scripts/dom-helpers.js';
+
+function createCard(cardData) {
+  let cardsContainer = document.querySelector('.cards-container');
+  const [
+    imageContainer,
+    imageAltText,
+    title,
+    desc,
+    titlelink,
+    btn,
+    btnLink,
+  ] = [...cardData.children];
+  const imgElement = imageContainer.querySelector('img');
+  imgElement.setAttribute('alt', imageAltText.textContent);
+  imageContainer.className = 'card-img';
+  title.className = 'title';
+  desc.className = 'desc';
+  btn.className = 'card-btn';
+
+  const RPCard = div({ class: 'rp-card' }, imageContainer, div({ class: 'text-content' }, a({ href: titlelink.textContent }, title), desc, a({ href: btnLink.textContent }, btn)));
+  titlelink.remove();
+  btnLink.remove();
+  imageAltText.remove();
+  if (!cardsContainer) {
+    cardsContainer = div({ class: 'cards-container' }, RPCard);
+  } else {
+    cardsContainer.append(RPCard);
+  }
+  document.querySelector('.research-publications-cards').append(cardsContainer);
+}
+
+export default async function decorate(block) {
+  const [title, button, link, rpFirstCard, rpSecondCard] = [...block.children];
```

Comment:
There can be `n` number of cards authored. Could you please handle that in your JS? 
```js
const [title, button, link, ...rpCards] = [...block.children];
```
like so..